packages/botonic-nlu/src/preprocessing.js

```diff
@@ -106,3 +106,55 @@ export function detectLang(input, langs) {
   }
   return res
 }
+
+export function parseUtterance(utterance) {
```

Comment:
I would call it st like "removeEntityType" , or add a comment describing what it does

Reply:
documented!